tools/test_configs/config_paths.json

```diff
@@ -8,5 +8,5 @@
     "ESP8266_WIFI" : "ESP8266Interface.json",
     "WICED_WIFI" : "WicedInterface.json",
     "ISM43362_WIFI" : "ISM43362Interface.json",
-    "IDW0XX1_WIFI" : "SpwfSAInterface.json"
+    "TEST_APP" : "test_app.json"
```

Comment:
Is this removal of IDW0XX1_WIFI config intentional? Also the "TEST_APP" could be something more informal if it relates only to the mactester.

Reply:
Probably by accident. Will rename config.